app/views/gobierto_participation/participation_events/_event.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="person_event-item">
+  <h3><a href="participation_agenda_event.html"><%= event.title %></a></h3>
+
+  <div class="meta">
```

Comment:
Could we hide this block with a `pending` block?

Reply:
https://github.com/PopulateTools/gobierto/pull/948/commits/86cef964afbe1ddbf6b2316675bdb8e0e9139e61